via/views/view_pdf.py

```diff
@@ -16,15 +23,33 @@
 )
 def view_pdf(request):
     """HTML page with client and the PDF embedded."""
-    nginx_server = request.registry.settings["nginx_server"]
-    pdf_url = QueryParams.build_url(
-        request.matchdict["pdf_url"], request.params, strip_via_params=True
-    )
 
     return {
-        "pdf_url": Markup(f"{nginx_server}/proxy/static/{pdf_url}"),
+        "pdf_url": Markup(_make_pdf_redirect(request)),
         "client_embed_url": Markup(request.registry.settings["client_embed_url"]),
-        "h_open_sidebar": asbool(request.params.get(QueryParams.OPEN_SIDEBAR, False)),
-        "h_request_config": request.params.get(QueryParams.CONFIG_FROM_FRAME, None),
         "static_url": request.static_url,
+        "hypothesis_config": _hypothesis_config(request),
     }
+
+
+def _hypothesis_config(request):
+    config = {"showHighlights": True, "appType": "via"}
```

Comment:
These two items moved from the front-end to here. The upside is the front end isn't trying to string concatenate it's way to JSON.